packages/fxa-content-server/app/images/icon-show-password-closed.svg

```diff
@@ -0,0 +1,9 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="55" height="37" viewBox="-10 -7 47 38" style="stroke:#0c0c0d">
```

Comment:
Had to use new svgs here because the input field in login behaves differently than input field in settings. The height and viewBox were adjusted as well as adding the stroke color.